docs/ops.html

```diff
@@ -3736,7 +3736,7 @@ <h4 class="anchor-heading"><a id="kraft_missing" class="anchor-link"></a><a href
   <h4 class="anchor-heading"><a id="kraft_zk_migration" class="anchor-link"></a><a href="#kraft_zk_migration">ZooKeeper to KRaft Migration</a></h4>
 
   <p>
-    <b>ZooKeeper to KRaft migration is considered an Early Access feature and is not recommended for production clusters.</b>
+    <b>Limitations:</b>
   </p>
 
   <p>The following features are not yet supported for ZK to KRaft migrations:</p>
```

Comment:
I also wonder if `Downgrading to ZooKeeper` is not possible yet or already supported.

Assuming all features of [KIP-866](https://cwiki.apache.org/confluence/display/KAFKA/KIP-866+ZooKeeper+to+KRaft+Migration) are covered, rollback is possible until we exit migration mode right?